pkg/remoteconfig/state/repository_test.go

```diff
@@ -96,6 +125,42 @@ func TestUpdateNewConfig(t *testing.T) {
 	assert.Equal(t, path, cached.Path)
 	assert.EqualValues(t, len(data), cached.Length)
 	assertHashesEqual(t, hashes, cached.Hashes)
+
+	// Do the same with the unverified repository, there should be no functional difference
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** ‏Just to make sure I get it, even without any stored root (ie incomplete TUF repo, which is not the same as invalid) the update should be accepted?